docs/docs/getting-started.md

```diff
@@ -41,9 +41,63 @@ $ npm install --save react react-dom babel-preset-react
 $ webpack
 ```
 
-> Note:
->
-> If you are using ES2015, you will want to also use the `babel-preset-es2015` package.
+Ensure you have a `.babelrc` including the react preset.
+
+A good starting webpack config for production mode would be:
+```javascript
+module.exports = {
+  entry: [
+    './client/index.jsx',
+  ],
+  output: {
+    path: path.join(__dirname, 'dist'),
+    filename: 'bundle.js',
+  },
+  resolve: {
+    modulesDirectories: ['node_modules', 'client'],
```

Comment:
I don't mind on this one, I use `jsx` for React components and then `js` for all pure JS files. The `client` convention as well is something I have used in a few projects. I am happy to change this.